src/about_alan.md

```diff
@@ -1,20 +1,19 @@
 # About `alan`
 
-`alan` is meant to fit like a glove. Like a glove, it's meant to be familiar and obvious how to use and generally not restricting. But like a glove, it has a few restrictions at the edges that going without it wouldn't have. And also like a glove, `alan` lets you grasp at things that would otherwise hurt you if you tried. ;)
+`alan` is a natively-parallel, statically-compiled, type-inferred language with a familiar syntax and many compile-time and run-time safety guarantees.
```

Comment:
😎

Reply:
![:sunglasses:](https://www.k9ofmine.com/wp-content/uploads/2017/09/sunglasses-for-dogs.jpg)